src/main/java/seedu/foodrem/logic/commands/itemcommands/SortCommand.java

```diff
@@ -31,10 +31,10 @@ public SortCommand(Comparator<Item> comparator) {
     }
 
     @Override
-    public CommandResult execute(Model model) {
+    public CommandResult<String> execute(Model model) {
         requireNonNull(model);
         model.updateSortedItemList(comparator);
-        return new CommandResult(String.format(Messages.MESSAGE_ITEMS_SORTED_OVERVIEW,
+        return CommandResult.from(String.format(Messages.MESSAGE_ITEMS_SORTED_OVERVIEW,
```

Comment:
Strangely, it seems some messages are from the Messages class, is there a reason for this discrepency? (Should this be refactored?)

Reply:
Yes, was looking to refactor this some time in the future. Could you help create an issue?